types/bootstrap/js/dist/popover.d.ts

```diff
@@ -0,0 +1,267 @@
+import * as Popper from 'popper.js';
+
+type ContentFunction = () => void;
```

Comment:
Does it make sense to make three different types for the same signature?

Reply:
I will remove it